standard/docs/en/getting_started/releases_and_records.md

```diff
@@ -23,7 +23,7 @@ There are two kinds of OCDS document:
 
 ### Releases
 
-Releases are immutable – presenting information about a particular event in the lifetime of a contracting process. Publishers must not edit a release after publication without changing the release `id`.
+Releases are immutable – presenting information about a particular event in the lifetime of a contracting process.
```

Comment:
I think it's important to keep the additional guidance to illustrate what immutable means in practical terms for OCDS publishers, I struggled to create a non-normative version of the current sentence, so suggesting an alternative:

> The release `id` needs to be changed whenever the value of another field in the release is updated.